components/NewPersonView/EventLink.tsx

```diff
@@ -4,13 +4,14 @@ import { Case } from 'types';
 import { generateInternalLink as generateLegacyUrl } from 'utils/urls';
 import s from './index.module.scss';
 
+const workflowsPilotUrl = `https://workflows-social-care-service.hackney.gov.uk`;
```

Comment:
Instead of hardcoding the URL for each environment we should use the hook we created for this, see https://github.com/LBHackney-IT/lbh-social-care-frontend/blob/82cad6b9e00359b61e5a8f82a661691e7df677d3/components/Dashboard/DashboardWrapper.tsx#L68 as an example for the code and https://github.com/LBHackney-IT/lbh-social-care-frontend/blob/82cad6b9e00359b61e5a8f82a661691e7df677d3/components/Dashboard/DashboardWrapper.spec.tsx#L37-L61 for test example.